app/models/topical_event.rb

```diff
@@ -110,11 +110,18 @@ def announcements
   def organisations
     @content_item.content_item_data.dig("links", "organisations")&.map do |organisation|
       {
+        content_id: organisation["content_id"],
         base_path: organisation["base_path"],
         title: organisation["title"],
         crest: organisation.dig("details", "logo", "crest"),
         brand: organisation.dig("details", "brand"),
       }
     end
   end
+
+  def emphasised_organisations
+    @content_item.content_item_data.dig("details", "emphasised_organisations")&.map do |organisation_id|
+      [] << organisation_id
+    end
+  end
```

Comment:
As far as I can see, this method is taking an array of content IDs and returning the same array.  Is there something I've missed?

Reply:
Oh, looks like it has been updated in the next commit.  Maybe it could just be introduced in the latter commit only?

Reply:
Also resolved through squashing